pkg/cmd/attestation/verification/tuf.go

```diff
@@ -26,15 +28,38 @@ func DefaultOptionsWithCacheSetting(tufMetadataDir o.Option[string]) *tuf.Option
 		opts.DisableLocalCache = true
 	}
 
-	// Set the cache path to the provided dir, or a directory owned by the CLI
-	opts.CachePath = tufMetadataDir.UnwrapOr(filepath.Join(config.CacheDir(), ".sigstore", "root"))
+	// During tests, use a temporary directory that will be cleaned up
+	if os.Getenv("GO_INTEGRATION_TEST") == "1" {
```

Comment:
This feels like the wrong way to do this. Is it not possible to set the CachePath via the opts instead?